strr-api/devops/vaults.gcp.env

```diff
@@ -5,6 +5,11 @@ AUTH_API_URL="op://API/$APP_ENV/auth-api/AUTH_API_URL"
 AUTH_API_VERSION="op://API/$APP_ENV/auth-api/AUTH_API_VERSION"
 LEGAL_API_URL="op://API/$APP_ENV/legal-api/LEGAL_API_URL"
 LEGAL_API_VERSION_2="op://API/$APP_ENV/legal-api/LEGAL_API_VERSION_2"
+LTSA_API_URL="op://API/$APP_ENV/ltsa/LTSA_API_URL"
```

Comment:
@pwei1018 We have some new env variables we need configured. I will message you on teams with the details.